homeassistant/components/litejet/scene.py

```diff
@@ -35,20 +39,22 @@ class LiteJetScene(Scene):
 
     def __init__(self, entry_id, lj: LiteJet, i, name):  # pylint: disable=invalid-name
         """Initialize the scene."""
-        self._entry_id = entry_id
         self._lj = lj
         self._index = i
-        self._name = name
+        self._attr_unique_id = f"{entry_id}_{i}"
+        self._attr_name = name
 
-    @property
-    def name(self):
-        """Return the name of the scene."""
-        return self._name
+    async def async_added_to_hass(self) -> None:
+        """Run when this Entity has been added to HA."""
+        self._lj.on_connected_changed(self._on_connected_changed)
 
-    @property
-    def unique_id(self):
-        """Return a unique identifier for this scene."""
-        return f"{self._entry_id}_{self._index}"
+    async def async_will_remove_from_hass(self) -> None:
+        """Entity being removed from hass."""
+        self._lj.unsubscribe(self._on_connected_changed)
+
+    def _on_connected_changed(self, connected: bool, reason: str) -> None:
+        self._attr_available = connected
+        self.schedule_update_ha_state()
```

Comment:
Since `LiteJetScene` does not implement `async_update`, you can just call `self.async_write_ha_state()` which will update the state immediately. That can be implemented in a separate PR though.